static/edit.js

```diff
@@ -0,0 +1,45 @@
+$(document).ready( function() {    
+    $('#edit_form').hide(); // form off
+    var id = $('article[class=single_entry]').attr("id");
+    event.preventDefault();
+    $.ajax({
+        url : "../edit",
```

Comment:
This is an odd looking URL.  You shouldn't require the `..` at the beginning if you provide an absolute URL like `/edit`.